hbase_sql/src/main/java/com/splicemachine/ddl/ZooKeeperDDLWatchChecker.java

```diff
@@ -46,6 +55,18 @@
     private Watcher changeIdWatcher;
     private final DDLZookeeperClient zkClient;
 
+    private static final InterProcessMutex lock;
+
+    static {
+        String ensemble = ZKConfig.getZKQuorumServersString(HConfiguration.unwrapDelegate());
+        CuratorFramework client = CuratorFrameworkFactory.newClient(ensemble, new ExponentialBackoffRetry(1000,3));
+        client.start();
+        String rootPath = SIDriver.driver().getConfiguration().getSpliceRootPath();
+        String ddlPath = rootPath+HConfiguration.DDL_PATH;
+        lock = new InterProcessMutex(client, ddlPath);
+    }
+
+
```

Comment:
I think this synchronization mechanism is too heavyweight. Could each RS persist it in its own ephemeral ZNode and when starting up we take the maximum demarcation point?